service/history/task/cross_cluster_task_processor.go

```diff
@@ -219,9 +225,13 @@ func (p *crossClusterTaskProcessor) processLoop() {
 		sw := p.metricsScope.StartTimer(metrics.CrossClusterFetchLatency)
 
 		var taskRequests []*types.CrossClusterTaskRequest
-		err := p.taskFetcher.Fetch(p.shard.GetShardID()).Get(context.Background(), &taskRequests)
```

Comment:
Do you mind creating a Jira task for this to examine all the context.Background() calls we have in the codebase?

Reply:
https://github.com/uber/cadence/issues/4664